spec/features/bug_tickets_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
+
+feature 'Bug tickets handling' do
+  describe 'autorized as user' do
+    let(:user) { create(:user) }
+    let(:bug_attr) { { title: 'Bug title', description: 'Bug description' } }
+
+    before { sign_in user }
+
+    context 'create bug ticket' do
+      scenario 'create bug ticket' do
+        visit new_bug_ticket_path
+        fill_in(I18n.t('bug_tickets.form.title'), with: bug_attr[:title])
```

Comment:
Same here about empty lines. Try use it and you we will see how your code beign more readable